DNN Platform/Library/Entities/Portals/PortalSettings.cs

```diff
@@ -201,6 +215,10 @@ public UserInfo UserInfo
             }
         }
 
+        /// <summary>
+        /// Gets the mode the user is viewing the page in.
+        /// </summary>
+        [Obsolete("Deprecated in v9.8.1, use Personalization.GetUserMode() instead, Scheduled for removal in v10.")]
         public Mode UserMode
         {
             get

```

Comment:
Should the body of this property just return `Personalization.GetUserMode()`, so we're not duplicating the implementation?

Reply:
Yep, forgot to do that, it is pushed in my latest commit. Thanks.